src/colorize/io.cr

```diff
@@ -164,11 +165,17 @@ module Colorize
   # It adds `#to_colorizable` method to create a new `ColorizableIO` instance.
   module IOExtension
     # Return `ColorizableIO` to colorize the output to `self`.
-    # If `self` is not `ColorizableIO`, it returns a new `IO` instance.
+    #
+    # If `self` is not `ColorizableIO`, it returns a new `IO` instance with *when_for_new* policy.
     # If `self` is `ColorizableIO` already, it returns itself.
-    def to_colorizable
-      return self if is_a?(ColorizableIO)
-      Colorize::IO.new self
+    def to_colorizable(when_for_new = When::Always)
+      Colorize::IO.new self, when_for_new
     end
   end
 end
+
+class IO::FileDescriptor
```

Comment:
I think it is just a matter of personal preference. But original `colorize.cr` extends `Object` class outside `ObjectExtension`, I followed it.